### DISCUSSION TITLE
Speeding up things

### DISCUSSION THREAD

**Discussion by alfredbuehler** [aljawaid/PluginManager#46] [General]
The page Manual Plugins is, at least on my systems, awfully slow.
We should gather ideas on how we can speed up the loading of that page.

- Those plugins don't get updated frequently, I assume. Maybe we could gather the information in a background job, and then use that information for building the page.

**Comment by aljawaid** [comment]
I dont have that problem, the manual plugins shows 17 and is fine in speed, same as installed plugins which is 38 and also fine in speed

Maybe the issue is local to you?

**Reply by alfredbuehler** [reply]
> Maybe the issue is local to you?

It depends on what you mean with “fine in speed”. But maybe you're right.  I'll keep it in mind.

**Reply by aljawaid** [reply]
I dont notice anything slow when I load them pages.... no delay compared to the rest of the site.  The directory takes a noticeable extra second I guess, but with 100+ plugins to show, I can ignore it.
